azext_iot/tests/iothub/devices/test_iot_edge_hierarchy_int.py

```diff
@@ -0,0 +1,216 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import pytest
+from azext_iot.tests.iothub import IoTLiveScenarioTest
+
+
+@pytest.mark.usefixtures("set_cwd")
+class TestNestedEdgeHierarchy(IoTLiveScenarioTest):
```

Comment:
Not strictly necessary for this PR, but at some point we need to converge on the new fixtures oriented test pattern (as introduced in ADU CLI, and applied in this in-flight PR https://github.com/Azure/azure-iot-cli-extension/pull/582)